web/libs/datamanager/src/components/CellViews/Annotators/Annotators.jsx

```diff
@@ -72,6 +80,79 @@ export const Annotators = (cell) => {
   );
 };
 
+export const InfiniteVariantSelect = observer(
```

Comment:
This component should probably be a a molecule in app-core (`UserSelect` or `UserDropdown` or similar) holding the extra logic of the user while rendering the select given we will need this component in Filters, DM action modals and more places potentially